src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealService.java

```diff
@@ -91,13 +94,21 @@ public Long submitAppeal(SyaCaseWrapper appeal, String userToken) {
         return caseDetails.getId();
     }
 
+    private void checkHasValidRole(IdamTokens idamTokens) {
```

Comment:
Minor: by convention this should be moved below with the other private methods

Reply:
Good spot, Done thanks